client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -29,7 +29,7 @@ import { isEnabled } from 'config';
 import { FEATURE_GOOGLE_ANALYTICS } from 'lib/plans/constants';
 
 const debug = debugFactory( 'calypso:my-sites:site-settings' );
-const googleAnalyticsCodeValidator = code => ! code || code.match( /^UA-\d+-\d+$/i );
+const googleAnalyticsCodeIsValid = code => ! code || code.match( /^UA-\d+-\d+$/i );
```

Comment:
why the name change? should we go ahead and give it a more _function_y name like `validateGoogleAnalyticsCode`?